From 8b965af7fe02057a9fa2049c32e4b5ba15cddb37 Mon Sep 17 00:00:00 2001
From: Richard Hu <richard.hu@technexion.com>
Date: Mon, 21 Nov 2022 16:50:43 +0800
Subject: [PATCH] fsl-imx-preferred-env.inc: change to use weak assignment for
 'PREFERRED_PROVIDER_virtual/kernel'

Make 'PREFERRED_PROVIDER_virtual/kernel' allow to be overwritten in local.conf.
---
 meta-sdk/conf/distro/include/fsl-imx-preferred-env.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta-sdk/conf/distro/include/fsl-imx-preferred-env.inc b/meta-sdk/conf/distro/include/fsl-imx-preferred-env.inc
index ac8b7a63a..efa6169d9 100644
--- a/meta-sdk/conf/distro/include/fsl-imx-preferred-env.inc
+++ b/meta-sdk/conf/distro/include/fsl-imx-preferred-env.inc
@@ -1,5 +1,5 @@
 PREFERRED_PROVIDER_virtual/bootloader_imx = "u-boot-imx"
-PREFERRED_PROVIDER_virtual/kernel_imx = "linux-imx"
+PREFERRED_PROVIDER_virtual/kernel_imx ?= "linux-imx"
 
 PREFERRED_VERSION_linux-libc-headers_imx ?= "5.10"
 
-- 
2.25.1

